tests/python/frontend/onnx/test_forward.py

```diff
@@ -2815,6 +2815,48 @@ def repeat(N, D):
 
 @tvm.testing.parametrize_targets
 def test_convtranspose(target, dev):
+    def verify_convtranspose_with_output_shape(
```

Comment:
Can you remove "test_convtranspose_output_shape", from `unsupported_onnx_tests`?